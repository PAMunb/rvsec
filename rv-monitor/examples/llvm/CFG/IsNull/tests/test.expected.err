isNull not called before deref!
isNull not called before deref!
