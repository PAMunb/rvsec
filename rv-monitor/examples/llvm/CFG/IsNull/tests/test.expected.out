sum: 15
sum: 15
